src/Skeleton/README.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Skeleton'
+type: 'component'
+components:
+- Skeleton
+categories:
+- Status & metadata
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+---
+
+``Skeleton`` component can be used directly in your components in place of content that is loading. 
+This tool has the ability to customize loading state of you component.
+You can read more about ``Skeleton`` on the [official documentation repository](https://github.com/dvtng/react-loading-skeleton).
+
+## Basic Usage
+
+For convenient customization of the loading component, **width** and **height** properties are available, 
+you can also set the number of columns in the displayed ``Skeleton``.
+
+```jsx live
+() => {
+  const [isChecked, setChecked] = useState(false);
+  const handleChange = e => setChecked(e.target.checked);
+
+  const exampleTitle = 'Example title';
+  const exampleDescription = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ' +
+          'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown ' +
+          'printer took a galley of type and scrambled it to make a type specimen book. It has survived not ' +
+          'only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.';
+
+  return (
+    <>
+      <Form.Checkbox 
+        className="mb-3"
+        checked={isChecked}
+        onChange={handleChange}
+      >
+        Show loading state
```

Comment:
I like the ability to toggle between loading / not loading, but am thinking it might make sense to always default it to show the loading state, since that's largely what a user would expect to see on this page right away.

Reply:
Compleated